deeprank_gnn/models/contact.py

```diff
@@ -25,15 +22,19 @@ def vanderwaals_potential(self) -> float:
 class ResidueContact(Contact):
     "A contact between two residues from a structure"
 
-    def __init__(self, residue1: Residue, residue2: Residue,
-                 distance: float,
-                 electrostatic_potential: float,
-                 vanderwaals_potential: float):
+    def __init__( # pylint: disable=too-many-arguments
+        self,
+        residue1: Residue,
+        residue2: Residue,
+        distance: float,
+        electrostatic_potential: float,
+        vanderwaals_potential: float,
+    ):
 
         self._residue1 = residue1
         self._residue2 = residue2
 
-        Pair.__init__(self, residue1, residue2)
+        # Pair.__init__(self, residue1, residue2)
```

Comment:
Is this line supposed to do anything?

Reply:
it initializes the parent class.

Reply:
Oh, I see. But the parent class of ResidueContact is Contact, not Pair, right? Pair is parent class for Contact instead. I think we should use `super()` (see [here](https://docs.python.org/3/library/functions.html#super)) both in Contact and ResidueContact `__init__` methods